docs/ko/docs/help-fastapi.md

```diff
@@ -1,162 +1,269 @@
 
-  GitHub에서 FastAPI를 "watch"할 수 있습니다 (오른쪽 상단 watch 버튼을 클릭): https://github.com/fastapi/fastapi. 👀
+---
 
-  "Releases only" 대신 "Watching"을 선택하면 다른 사용자가 새로운 issue를 생성할 때 알림이 수신됩니다.
+다른 사람들의 질문 (디스커션 또는 이슈에서) 해결을 도울 수 있는 방법은 다음과 같습니다.
 
-  그런 다음 이런 issues를 해결 할 수 있도록 도움을 줄 수 있습니다.
+### 질문 이해하기
 
-  ## 이슈 생성하기
+* 질문하는 사람이 가진 **목적**과 사용 사례를 이해할 수 있는지 확인하세요.
 
-  GitHub 저장소에 [새로운 이슈 생성](https://github.com/fastapi/fastapi/issues/new/choose) 을 할 수 있습니다, 예를들면 다음과 같습니다:
+* 질문 (대부분은 질문입니다)이 **명확**한지 확인하세요.
 
-  - **질문**을 하거나 **문제**에 대해 질문합니다.
-  - 새로운 **기능**을 제안 합니다.
+* 많은 경우, 사용자가 가정한 해결책에 대한 질문을 하지만, 더 **좋은** 해결책이 있을 수 있습니다. 문제와 사용 사례를 더 잘 이해하면 더 나은 **대안적인 해결책**을 제안할 수 있습니다.
 
-  **참고**: 만약 이슈를 생성한다면, 저는 여러분에게 다른 사람들을 도와달라고 부탁할 것입니다. 😉
+* 질문을 이해할 수 없다면, 더 **자세한 정보**를 요청하세요.
 
-  ## Pull Request를 만드십시오
+### 문제 재현하기
 
-  Pull Requests를 이용하여 소스코드에 [컨트리뷰트](https://github.com/fastapi/fastapi/blob/master/docs/en/docs/contributing.md){.internal-link target=_blank} 할 수 있습니다. 예를 들면 다음과 같습니다:
+대부분의 경우, 질문은 질문자의 **원본 코드**와 관련이 있습니다.
 
-  - 문서에서 찾은 오타를 수정할 때.
+많은 경우, 코드의 일부만 복사해서 올리지만, 그것만으로는 **문제를 재현**하기에 충분하지 않습니다.
 
-  - FastAPI를 [편집하여](https://github.com/fastapi/fastapi/edit/master/docs/en/data/external_links.yml) 작성했거나 찾은 문서, 비디오 또는 팟캐스트를 공유할 때.
+* 질문자에게 <a href="https://stackoverflow.com/help/minimal-reproducible-example" class="external-link" target="_blank">최소한의 재현 가능한 예제</a>를 제공해달라고 요청하세요. 이렇게 하면 코드를 **복사-붙여넣기**하여 직접 실행하고, 동일한 오류나 동작을 확인하거나 사용 사례를 더 잘 이해할 수 있습니다.
 
-    - 해당 섹션의 시작 부분에 링크를 추가했는지 확인하십시오.
+* 너그러운 마음이 든다면, 문제 설명만을 기반으로 직접 **예제를 만들어**볼 수도 있습니다. 하지만, 이는 시간이 많이 걸릴 수 있으므로, 먼저 질문을 명확히 해달라고 요청하는 것이 좋습니다.
 
-  - 당신의 언어로 [문서 번역하는데](https://github.com/fastapi/fastapi/blob/master/docs/en/docs/contributing.md#translations){.internal-link target=_blank} 기여할 때.
+### 해결책 제안하기
 
-    - 또한 다른 사용자가 만든 번역을 검토하는데 도움을 줄 수도 있습니다.
+* 질문을 충분히 이해한 후에는 가능한 **답변**을 제공할 수 있습니다.
 
-  - 새로운 문서의 섹션을 제안할 때.
+* 많은 경우, 질문자의 **근본적인 문제나 사용 사례**를 이해하는 것이 중요합니다. 그들이 시도하는 방법보다 더 나은 해결책이 있을 수 있기 때문입니다.
 
-  - 기존 문제/버그를 수정할 때.
+### 해결 요청하기
 
-  - 새로운 feature를 추가할 때.
+질문자가 답변을 확인하고 나면, 당신이 문제를 해결했을 가능성이 높습니다. 축하합니다, **당신은 영웅입니다**! 🦸
 
-  ## 채팅에 참여하십시오
+* 이제 문제를 해결했다면, 질문자에게 다음을 요청할 수 있습니다.
 
-  👥 [디스코드 채팅 서버](https://discord.gg/VQjSZaeJmf) 👥 에 가입하고 FastAPI 커뮤니티에서 다른 사람들과 어울리세요.
+    * GitHub 디스커션에서: 댓글을 **답변**으로 표시하도록 요청하세요.
+    * GitHub 이슈에서: 이슈를 **닫아달라고** 요청하세요.
 
-  /// tip
+## GitHub 저장소 보기
 
-  질문이 있는 경우, [GitHub 이슈 ](https://github.com/fastapi/fastapi/issues/new/choose) 에서 질문하십시오, [FastAPI 전문가](https://github.com/fastapi/fastapi/blob/master/docs/en/docs/fastapi-people.md#experts) 의 도움을 받을 가능성이 높습니다{.internal-link target=_blank} .
+GitHub에서 FastAPI를 "watch"할 수 있습니다 (오른쪽 상단 watch 버튼을 클릭): <a href="https://github.com/fastapi/fastapi" class="external-link" target="_blank">https://github.com/fastapi/fastapi</a>. 👀
 
-  ///
+"Releases only" 대신 "Watching"을 선택하면, 새로운 issue나 질문이 생성될 때 알림을 받을 수 있습니다. 또한, 특정하게 새로운 issue, discussion, PR 등만 알림 받도록 설정할 수도 있습니다.
```

Comment:
```suggestion
"Releases only" 대신 "Watching"을 선택하면, 새로운 이슈나 질문이 생성될 때 알림을 받을 수 있습니다. 또한, 특정하게 새로운 이슈, 디스커션, PR 등만 알림 받도록 설정할 수도 있습니다.
```
I think It would be better to translate "issue"→"이슈" and "discussion"→"디스커션" in lines 130, 132.
Because "이슈" and "디스커션" are already used in lines 72, 89, and 123.